### DISCUSSION TITLE
Feature Request: Upload files via remote URL

### DISCUSSION THREAD

**Discussion by TheHamkerCat** [filebrowser/filebrowser#1617] [Ideas]
**Is your feature request related to a problem? Please describe.**
I want to download files to my file server using a URL (filebrowser will download the file from the provided URL).

**Describe the solution you'd like**
Add another option in **Upload**, Upload via URL, instead of opening file chooser directly.

**Describe alternatives you've considered**
Currently, I have to ssh to my file server and then use wget to download the file via that URL.

**Comment by ramiresviana** [comment]
The command shell allows users to run commands in the FileBrowser interface. Adding wget to the user commands allows it to  download files into the server.

**Comment by TheHamkerCat** [comment]
> The command shell allows users to run commands in the FileBrowser interface. Adding wget to the user commands allows it to download files into the server.

But it would be great to have it with the "Upload file" button

**Comment by TheHamkerCat** [comment]
Closed the issue, Great!

**Comment by ItsNoted** [comment]
The wget command works but it does not show progress of the download. It just downloads the file with no progress and freezes the current command screen. I had to open a new tab to view the files and refresh to see the file size growing in size. Not ideal but it works.
